### PR TITLE
miniserve: add shell completions

### PR DESCRIPTION
**Pull request by moonfruit**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/miniserve.rb [hunk 1]
```diff
@@ -16,6 +17,14 @@ class Miniserve < Formula
...
   def install
     system "cargo", "install", *std_cargo_args
+
+    system "#{bin}/miniserve --print-completions bash > miniserve.bash"
+    system "#{bin}/miniserve --print-completions fish > miniserve.fish"
+    system "#{bin}/miniserve --print-completions zsh > _miniserve"
+
+    bash_completion.install "miniserve.bash"
+    fish_completion.install "miniserve.fish"
+    zsh_completion.install "_miniserve"
```
Comment (carlocab) [inline Formula/miniserve.rb:27]: Can we do this more like how `deno` (for example) does it?

Comment (moonfruit) [reply Formula/miniserve.rb]: OK.

**Comment by moonfruit** [comment]
I found a lot of formulae that want to redirect command standard output to a file. Maybe we should add a util function to Utils to do this.

**Comment by carlocab** [comment]
Possibly. Would you like to open a PR for that in Homebrew/brew?

**Review by carlocab — APPROVED** [review]
Thanks, @moonfruit.

**Pull request closed without merging**